views/products/demo/at2.tpl

```diff
@@ -6,18 +6,59 @@
     source code of the project.
     If you're interested, please contact us, and we'll get you in contact
     with the original authors.</p>
+<p>Here are some screenshots of the demo:</p>
+<div class="columns">
+    <div class="click-zoom">
+        <label>
+            <input type="checkbox">
+            <img src="../../../resources/products/images/at2/at2-demo-1.png">
+            <br>The introduction of the demo with a short explanation of what
+            it is about.
+        </label>
+    </div>
+    <div class="click-zoom">
+        <label>
+            <input type="checkbox">
+            <img src="../../../resources/products/images/at2/at2-demo-2.png">
+            <br>On the next screen it shows the account creation and the
+            network of the nodes.
+        </label>
+    </div>
+</div>
+<div class="columns">
+    <div class="click-zoom">
+        <label>
+            <input type="checkbox">
+            <img src="../../../resources/products/images/at2/at2-demo-3.png">
+            <br>Now you can transfer some assets to one of the other accounts
+            on the network. For this test you have 100k coins of whatever value
+            (currently 0US$ :)
```

Comment:
```suggestion
            (currently 0US$ :))
```

Reply:
Personal preference: I'm used to close the parentheses with a smiley...